src/lib/pipeline_configuration.py

```diff
@@ -57,15 +59,15 @@ def __init__(self, raw_field, coded_field, coda_filename, cleaner=None, code_sch
 class PipelineConfiguration(object):  
     DEV_MODE = True
 
-    PROJECT_START_DATE = isoparse("")
-    PROJECT_END_DATE = isoparse("")
+    PROJECT_START_DATE = isoparse("2019-02-04T00:00:00+0300") #To change before production time
```

Comment:
Put this comment on the preceding line and start it with "TODO".

Then it becomes easier to see when scanning the file, and you get editor support

Reply:
Might be worth setting the start/end dates to datetime.min/.max to make it more obvious that these dates are temporary

Reply:
Nice tips thanks alot